src/controllers/notifications.controllers.js

```diff
@@ -0,0 +1,50 @@
+import Notifications from "../models/Notifications.js";
+
+// Get notifications by user
+export const getNotificationsByUserId = async (req, res) => {
+  try {
+    const userId = req.params.user_id;
+    const notifications = await Notifications.find({ user_id: userId });
+
+    if (!notifications || notifications.length === 0) {
+      return res
+        .status(404)
+        .json({ message: "No notifications found for this user" });
+    }
+
+    res.status(200).json(notifications);
```

Comment:
follow the convention response:

success
res.status(200).json({ data: response, error: null });
 
 
Error
res.status(500).json({ data: null, error: error });

Reply:
changed as suggested